style.css

```diff
@@ -1,887 +1,858 @@
+}
+
+#workscontainer .container .work-pic-1 {
+  transition: all 0.2s ease-in-out;
+}
+
+#workscontainer .work-pic-1:hover {
+  transform: scale(1.05);
+}
+
+#workscontainer .container .work-description {
+  width: 100%;
+  height: auto;
+  margin-top: 15px;
+}
+
+#workscontainer .container .work-description h4 {
+  font-weight: bold;
   font-size: 32px;
   line-height: 44px;
+  font-family: 'Poppins', sans-serif;
   color: #172b4d;
 }
 
-.work-title {
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 600;
-  font-size: 13px;
-  line-height: 16px;
+#workscontainer .container .description-element {
+  width: 90%;
+  margin-top: 15px;
   display: flex;
   justify-content: space-between;
-  width: 80%;
-  margin: 15px 0;
+  align-items: center;
 }
 
-.client {
-  color: #344563;
+#workscontainer .container .work-description .description-element .element {
+  font-family: "Poppins", sans-serif;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: rgb(116 129 153);
+  list-style: none;
 }
 
-.dim {
-  color: #7a869a;
+.bullet {
+  width: 10px;
 }
 
-.project-desc {
+#workscontainer .container .work-description .description-text {
   font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 400;
+  margin: 20px auto;
+  font-weight: normal;
   font-size: 15px;
   line-height: 24px;
-  color: #344563;
-  margin: 10px 10px 10px 0;
+  color: rgb(52 69 99 / 100%);
+}
+
+#workscontainer .container .work-description .languages {
+  width: 80%;
+  list-style: none;
 }
 
-.langs {
+.lang {
+  float: left;
+  margin-right: 14px;
+  padding: 0 10px;
+  height: 24px;
+  font-family: "Poppins", sans-serif;
   display: flex;
-  flex-direction: row;
+  justify-content: center;
   align-items: center;
-  gap: 8px;
+  border-radius: 8px;
+  color: #6070ff;
+  background: #ebebff;
+  margin-bottom: 10px;
 }
 
-.tag {
+#workscontainer .container .work-description .languages .lang1 {
+  width: 50px;
+  height: 24px;
   font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 12px;
-  line-height: 16px;
-  letter-spacing: 0.03em;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 8px;
   color: #6070ff;
   background: #ebebff;
-  border-radius: 8px;
+}
+
+#workscontainer .container .work-description .languages .lang2 {
+  width: 95px;
+  height: 24px;
+  font-family: "Poppins", sans-serif;
   display: flex;
-  flex-direction: row;
+  justify-content: center;
   align-items: center;
-  padding: 4px 20px;
+  border-radius: 8px;
+  color: #6070ff;
+  background: #ebebff;
+}
+
+@media screen and (max-width: 991px) {
+  .nav-items {
+    position: fixed;
+    height: 100vh;
+    width: 100%;
+    right: 0;
+    top: 0;
+    font-size: 35px;
+    padding: 100px 0;
+    flex-direction: column;
+    justify-content: start;
+    align-items: flex-start;
+    background-color: rgb(13 35 235 / 77.4%);
+    transition: right 0.3s ease;
```

Comment:
- [ ] Kindly try to stick to the Figma background color for the mobile menu, so please try to use this background color ```#6070FF``` instead of ```rgb(13 35 235 / 77.4%)``` to be like the Figma Design 😊
- [ ] Also please try to decrease the distance between the icon and the link to be like the Figma design 😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/224786235-39c7bda3-0f9b-4778-867b-78196059ef2e.png)    |   ![image](https://user-images.githubusercontent.com/83360636/224786311-8765ec3b-4784-457f-b9cb-65a9e4a1f287.png)  |